invenio_drafts_resources/services/records/config.py

```diff
@@ -126,6 +127,9 @@ class RecordServiceConfig(RecordServiceConfigBase):
     # WHY: We want to force user input choice here.
     draft_cls = None
 
+    draft_indexer_cls = RecordIndexer
```

Comment:
note to self: mirroring the record indexer cls, see [here](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/config.py#L63)